<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Cad_Usuario</class>
 <widget class="QFrame" name="Form_Cad_Usuario">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LC INFORMÁTICA | CADASTRO USUÁRIOS</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../img/adduser_.png</normaloff>../../img/adduser_.png</iconset>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QFrame" name="fr_FormProdutos">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-1</y>
     <width>1000</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFF;
border: none</string>
   </property>
   <widget class="QLabel" name="lb_FormProdutos">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>10</y>
      <width>961</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 14px;
font-family: &quot;Arial&quot;;
font-weight: bold;

border-bottom: 2px solid #A2A2A2
}</string>
    </property>
    <property name="text">
     <string>CADASTRO USUÁRIO</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FotoProduto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>241</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #A2A2A2;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormProdutos_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>NOME COMPLETO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_nome">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>85</y>
      <width>450</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;
}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="placeholderText">
     <string>Digite seu nome</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormProdutos_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>290</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>NÍVEL DE ACESSO</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_Nivel_Acesso">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>320</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;

}
QComboBox:Focus {
border: 1px solid red;
}
 QComboBox::drop-down {
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 25px;
     border-left-width: 1px;
     border-left-color: darkgray;
     border-left-style: solid; /* just a single line */
     border-top-right-radius: 3px; /* same radius as the QComboBox */
     border-bottom-right-radius: 3px;
 }

 
</string>
    </property>
    <item>
     <property name="text">
      <string>SELECIONE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ADMINISTRADOR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>USUÁRIO</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lb_FormProdutos_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>290</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>PERMISSÕES</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_Permissoes">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>320</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;

}
QComboBox:Focus {
border: 1px solid red;
}
 QComboBox::drop-down {
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 25px;
     border-left-width: 1px;
     border-left-color: darkgray;
     border-left-style: solid; /* just a single line */
     border-top-right-radius: 3px; /* same radius as the QComboBox */
     border-bottom-right-radius: 3px;
 }
</string>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAtBottom</enum>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>SELECIONE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TODAS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VISUALIZAR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EDITAR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EXCLUIR</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lb_FormProdutos_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>960</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 14px;
font-family: &quot;Arial&quot;;
font-weight: normal;

border-bottom: 2px solid #A2A2A2;
color: #797979
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="fr_BotoesFormProdutos">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>1000</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: none;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormProdutos_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>NOME DE USUÁRIO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_Login">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>450</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;
}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="placeholderText">
     <string>Digite seu login</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormProdutos_6">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>210</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>SENHA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_Senha">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;

}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Digite sua senha</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormProdutos_9">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>210</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>REPITA A SENHA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_Senha2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>240</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;
}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Digite sua senha novamente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_SalvarUsuario">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>440</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #7AB32E;
color: #FFF
 }
QPushButton:hover{
background-color: #40a286
}</string>
    </property>
    <property name="text">
     <string>SALVAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/salvar/salvar.png</normaloff>:/salvar/salvar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Bt_CancelarUsuario">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>440</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #1E87F0;
color: #FFF
 }
QPushButton:hover{
background-color: #40a286
}</string>
    </property>
    <property name="text">
     <string>CANCELAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/sair/delete_delete_exit_1577.png</normaloff>:/sair/delete_delete_exit_1577.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>320</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>320</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tx_nome</tabstop>
  <tabstop>tx_Login</tabstop>
  <tabstop>tx_Senha</tabstop>
  <tabstop>tx_Senha2</tabstop>
  <tabstop>cb_Nivel_Acesso</tabstop>
  <tabstop>cb_Permissoes</tabstop>
 </tabstops>
 <resources>
  <include location="../Icones/icones.qrc"/>
 </resources>
 <connections/>
</ui>
